File: srm180/DinkyFish.java
import java.lang.*;

public class DinkyFish {

	public int monthsUntilCrowded(int tankVolume, int maleNum, int femaleNum) {
		int maxPopulation = tankVolume * 2;
		int currentPopulation = maleNum + femaleNum;
		int months = 0;
		while (currentPopulation <= maxPopulation) {
			int tmp = Math.min(maleNum, femaleNum);
			maleNum += tmp;
			femaleNum += tmp;
			currentPopulation = (maleNum + femaleNum);
			months += 1;
		}
				
		return months;
	}
}

File: srm180/dinky_fish.py
class DinkyFish:
	def monthsUntilCrowded(self, tankVolume, maleNum, femaleNum):
		max_population = tankVolume * 2
		current_population = maleNum + femaleNum
		months = 0
		while current_population <= max_population:
			tmp = min(maleNum, femaleNum)
			maleNum += tmp
			femaleNum += tmp
			current_population = (maleNum + femaleNum)
			months += 1
		
		return months